### PR TITLE
[ZEPPELIN-5374] Don't update paragraph config when latest checkpoint of flink is unchanged

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Simple PR to check whether the current latest checkpoint is changed, only update paragraph config when it is changed. 

### What type of PR is it?
[Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5374

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: flink/interpreter/src/main/java/org/apache/zeppelin/flink/JobManager.java [hunk 1]
```diff
@@ -254,11 +255,12 @@ public void run() {
               if (completedObject.has("external_path")) {
                 String checkpointPath = completedObject.getString("external_path");
                 LOGGER.debug("Latest checkpoint path: {}", checkpointPath);
-                if (!StringUtils.isBlank(checkpointPath)) {
+                if (!StringUtils.isBlank(checkpointPath) && !checkpointPath.equals(latestCheckpointPath)) {
```
Comment (cuspymd) [inline flink/interpreter/src/main/java/org/apache/zeppelin/flink/JobManager.java:258]: It seems that using the operator "not (!)" only once is good for readability.
```suggestion
                if (!(StringUtils.isBlank(checkpointPath) || checkpointPath.equals(latestCheckpointPath))) {
```

Comment (zjffdu) [reply flink/interpreter/src/main/java/org/apache/zeppelin/flink/JobManager.java:258]: Actually here 2 not is more readable. The if condition is when checkpoint is not null and it is not the same as latestCheckpointPath. 
If I change it to 1 not, it is a little harder to understand what the if condition means

**Review by cuspymd — APPROVED** [review]
Looks good!!

**Comment by zjffdu** [comment]
Will merge if no more comment

**Pull request closed without merging**